GhidraBuild/IDAPro/Python/9xx/loaders/xml_loader.py

```
@@ -0,0 +1,108 @@
+        status = xml.import_xml()
+    except idaxml.Cancelled:
+        msg = "XML PROGRAM import cancelled!"
+        print("\n" + msg)
+        idc.warning(msg)
+    except idaxml.MultipleAddressSpacesNotSupported:
+        msg  = "XML Import cancelled!"
+        msg += "\n\nXML Import does not currently support"
+        msg += "\nimporting multiple address spaces."
+        print("\n" + msg)
+        idc.warning(msg)
+    except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)